app/assets/javascripts/components/schools/schools_page.es6.jsx

```diff
@@ -47,7 +45,7 @@ class SchoolsPage extends Component {
         <div style={this.styles.container}>
           <Sidebar shouldShow={this.state.sidebar} />
           <div style={this.styles.body}>
-            <SchoolsGrid {...this.props} />
+            <SchoolsGrid schools={this.state.schools} />
```

Comment:
Note that we pass down STATE from this top-level component now - all child component are unaware of this and still will pass down props.